omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -44,7 +44,7 @@ def first_run?
   end
 
   def secrets_exists?
-    PrivateChef.existing_secrets.length > 0
+    File.exist?("/etc/opscode/private-chef-secrets.json")
```

Comment:
The reason we checked for non-zero length was that we had a situation  where an empty file existed, which stopped the installation from proceeding because of the preflight check.  Why the change to only `exist`?